homeassistant/components/light/isy994.py

```diff
@@ -69,6 +69,10 @@ def turn_on(self, brightness=100, **kwargs) -> None:
             _LOGGER.debug('Unable to turn on light.')
 
     @property
+    def state_attributes(self):
+        return {ATTR_BRIGHTNESS: self.value}
+        
```

Comment:
Looks like you have a line with some spaces in it.  Clear those so its just a blank line.